src/gui/mzroll/mainwindow.cpp

```diff
@@ -1731,6 +1731,56 @@ void MainWindow::open()
         Q_UNUSED(continueButton);
     }
 
+    int sampleCount = 0;
+    QStringList sampleList;
+    int projectCount  = 0;
+    QStringList projectList;
+    Q_FOREACH (QString filename, filelist) {
+        if (fileLoader->isEmdbProject(filename) 
+            || fileLoader->isMzrollDbProject(filename)
+            || fileLoader->isMzRollProject(filename)) {
+                projectCount++;
+                projectList << filename;
+            } else {
+                sampleCount++;
+                sampleList << filename;
+            }
+    }
+    filelist.clear();
+    if (sampleCount && !projectCount) {
+        filelist = sampleList;
+    } else if (!sampleCount && projectCount) {
+        if (projectCount > 1) {
+            filelist << projectList[0];
+        } else {
+            filelist = projectList;
+        }
+    } else {
+        QMessageBox msgBox;
+        msgBox.setText("It seems you are trying to load sample files and "
+                       "project files on one instance.\n\n"
+                       "Instead, you can choose to upload either of the two.");
+        QPushButton* sampleChosen = msgBox.addButton("Sample Files",
+                                                       QMessageBox::ActionRole);
+        QPushButton* projectChosen = msgBox.addButton("ProjectFiles",
+                                                     QMessageBox::RejectRole);
+
+        msgBox.setDefaultButton(sampleChosen);
```

Comment:
I would have made the project button as default. The first project in the file list.